radixdlt-java/src/main/java/com/radixdlt/client/atommodel/Identifiable.java

```diff
@@ -1,10 +1,10 @@
 package com.radixdlt.client.atommodel;
 
-import com.radixdlt.client.core.atoms.particles.RadixResourceIdentifer;
+import com.radixdlt.client.core.atoms.particles.RRI;
 
 /**
  * Temporary interface for representing a particle which is indexed at an RRI
```

Comment:
Is this still temporary?

Reply:
Yes, this should be removed down the line